src/current/_includes/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,178 @@
+These essential CockroachDB metrics enable you to build custom dashboards with tools, such as: 
+{% if include.deployment == 'self-hosted' %}
+* [Grafana](monitor-cockroachdb-with-prometheus.html#step-5-visualize-metrics-in-grafana)
+* [Datadog Integration](datadog.html) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics) column lists the corresponding Datadog metric which requires the `cockroachdb.` prefix. A given metric is not available using the Datadog Integration if the **Datadog Integration Metric Name** column is blank (such as for the `sys.cpu.host.combined.percent-normalized` metric).
+{% elsif include.deployment == 'dedicated' %}
+* [Datadog integration](../cockroachcloud/tools-page.html#monitor-cockroachdb-dedicated-with-datadog) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics) column lists the corresponding Datadog metric which requires the `crdb_dedicated.` prefix. A given metric is not available using the Datadog Integration if the **Datadog Integration Metric Name** column is not shown (such as for Row-Level TTL, Backup and Restore, and Table Statistics categories) or is blank (such as for the `sys.cpu.host.combined.percent-normalized` metric).
+* [Metrics export](../cockroachcloud/export-metrics.html)
+{% endif %}
+  
+The metrics are important to visualize in a custom dashboard. The **Usage** column explains how to make both practical and actionable use of the metric in a production deployment.
+
+## Platform
```

Comment:
I wonder if we can make the description column bigger; it's pretty squished in the preview

Reply:
Updated